Sources/GoogleMapsUtils/Clustering/Algorithms/SimpleCluster/GMUSimpleClusterAlgorithm1.swift

```diff
@@ -0,0 +1,74 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+/// TO-DO: Rename the class to `GMUSimpleClusterAlgorithm` once the linking is done and remove the objective c class.
+/// `GMUSimpleClusterAlgorithm` is a basic clustering algorithm that groups a set of `GMUClusterItem` objects into a fixed number of clusters (default 10).
```

Comment:
Let's stay consistent in our refdocs; other doc comments I've seen don't repeat the class name in the description, so we don't need "`GMUSimpleClusterAlgortihm` is". Can just start with "A basic clustering algorithm..."